debian/rules

```diff
@@ -69,3 +69,5 @@ override_dh_strip:
 # Upstream encourages the use of 'build' as the developer build output
 # directory, allow that directory to be present and still allow dh to work.
 .PHONY: build
+build:
```

Comment:
Oh, you are right. I wasn't aware that the phony target do not consider implicit rules (like pattern rules). Then I recommend to move the .PHONY and build target up (just below the pattern rule).